css-isolation/app1/src/index.js

```diff
@@ -0,0 +1,2 @@
+import bootstrap from './bootstrap';
```

Comment:
Can you rather use dynamic import. Bundle loader implementation is deprecated. Or use dependOn in entrypoint advanced configs

Reply:
I must have copied it like this from one of the older examples. Changing this to a dynamic import causes a very weird bug I've never seen before. If I just change it it breaks the app. If I add `library: { type: 'var', name: 'app1' },` to the config it breaks with the following message `Uncaught SyntaxError: illegal character U+0040` which would be the character `@` from this portion of the bundled app1 `module.exports = app2@//localhost:3002/remoteEntry.js;`. If I remove the `@` it breaks with `Uncaught SyntaxError: illegal character :`. I am out of ideas because I have used the dynamic import syntax before and I have never seen this happen.

Reply:
I fixed it by removing bundle-loader from app1 webpack config. It was specifically loading bootstrap.js for some reason and it doesn't work when it's dynamically imported in index.js. Probably should do the same fix in the other examples that have this problem.